File: Pattern/pattern5/pattern.go
package main

import "fmt"

func pattern(x int) {
	for i := x; i >= 0; i-- {
		for j := 1; j <= i; j++ {
			fmt.Print("*")
		}
		fmt.Println("")
	}
}

func main() {
	pattern(8)
}

File: Pattern/pattern8/pattern.js
const pattern = (n) => {
  for (let i = 0; i < n; i++) {
    let line = "";
    for (let j = 1; j <= i; j++) {
      line += " ";
    }
    for (let k = 1; k <= n - i; k++) {
      line += "* ";
    }
    console.log(line);
  }
};

pattern(5);
